apps/linode-marketplace-openlitespeed-django/roles/post/templates/motd.j2

```diff
@@ -0,0 +1,9 @@
+*********************************************************
+Akamai Connected Cloud OpenLiteSpeed Django Marketplace App
+App URL: 
+* The sample Django site: http://{{ ansible_default_ipv4.address }}/
+* The sample Django admin page: http://{{ ansible_default_ipv4.address }}/admin
+Credentials File: /root/.credentials
```

Comment:
creds are now located in /home/$username/.credentials

Reply:
Check.